bower.json

```diff
@@ -40,7 +40,7 @@
     "patternfly": ">=3.29.3 <4.0.0",
     "jquery": "~3.2.1",
     "lodash": "~4.17.4",
-    "origin-web-common": "~0.0.70",
+    "origin-web-common": "v3.9.0-alpha.1",
```

Comment:
I wonder if we can still specify a range for prerelease versions. Does something like `>=v3.9.0-alpha.1 <3.10.0` work?

Reply:
Seems to work for npm at least testing with https://semver.npmjs.com/